src/components/Frame/Frame.tsx

```diff
@@ -262,15 +261,6 @@ class FrameInner extends PureComponent<CombinedProps, State> {
         />
       ) : null;
 
-    const skipToMainContentTarget = skipToContentTarget ? null : (
-      // eslint-disable-next-line jsx-a11y/anchor-is-valid
-      <a
-        id={APP_FRAME_MAIN_ANCHOR_TARGET}
-        ref={this.skipToMainContentTargetNode}
-        tabIndex={-1}
-      />
-    );
```

Comment:
Removing this would be a bit of a breaking change. Are you sure we don't need this anymore?

Reply:
Yeah this seems half-done

Either: there's still a need for a skipToMainContentTargetNode prop - in which case we need to do something with iit OR it's not needed anymore, in which case we need to comment on the prop saying it's deprecated and docuementation detailing that too., 
 

What happens if a users specifies a `skipToContentTarget` prop - we no longer emmbed that an.

Reply:
From what I can tell this is only used as a target for Skip To Content. I did a global search for `AppFrameMainContent` and only see it referenced in Polaris and GlobalNav (which I updated here https://github.com/Shopify/global-nav/pull/859 as well).
However with that said please let me know if there are other places I should check.

Reply:
Also the reason this was removed is because focus was still being given to the anchor since it's technically a valid focusable element within `<main>`. So by remove this it should go to a visible element in the content.

Reply:
@BPScott yes I left the prop since it was needed according to this PR https://github.com/Shopify/polaris-react/pull/2080

Reply:
Reading through this I think this change is fine. `skipToContentTarget` either points towards `APP_FRAME_MAIN` or the provided ref prop. This removes the private `skipToMainContentTargetNode` and the link that is rendered. Is the risk that someone incorrectly referenced that element?